src/Data/Foldable.purs

```diff
@@ -169,6 +170,10 @@ instance foldableMultiplicative :: Foldable Multiplicative where
 fold :: forall f m. Foldable f => Monoid m => f m -> m
 fold = foldMap id
 
+-- | Similar to 'foldl', but the result is encapsulated in a monad. 
```

Comment:
I think it's fine as-is, with an updated comment. Maybe we could provide a `foldMRec` too later, although I'm not sure that's even implementable with the same signature.